lua/lspconfig/jdtls.lua

```diff
@@ -129,26 +140,50 @@ configs[server_name] = {
   docs = {
     package_json = 'https://raw.githubusercontent.com/redhat-developer/vscode-java/master/package.json',
     description = [[
-
 https://projects.eclipse.org/projects/eclipse.jdt.ls
 
 Language server for Java.
 
-See project page for installation instructions.
+IMPORTANT: If you want all the features jdtls has to offer, [nvim-jdtls](https://github.com/mfussenegger/nvim-jdtls)
+is highly recommended. If all you need is diagnostics, completion, imports, gotos and formatting and some code actions
+you can keep reading here.
+
+For manual installation you can download precompiled binaries from the
+[official downloads site](http://download.eclipse.org/jdtls/snapshots/?d)
 
-Due to the nature of java, the settings for eclipse jdtls cannot be automatically
-inferred. Please set the following environmental variables to match your installation. You can set these locally for your project with the help of [direnv](https://github.com/direnv/direnv). Note version numbers will change depending on your project's version of java, your version of eclipse, and in the case of JDTLS_CONFIG, your OS.
+Due to the nature of java settings cannot be inferred, please set the following
```

Comment:
You removed a comma that should probably not have been removed

Reply:
Fixed